detd/scheduler.py

```diff
@@ -384,3 +385,50 @@ def reschedule(self):
 
             self.schedule.add_best_effort_padding(self.traffics[0])
             # FIXME: error handling
+
+class DataPath(enum.Enum):
+    AF_PACKET = 1
+    AF_XDP_ZC = 2
+
+class TxCapability(enum.Enum):
+    none = 0
+    Qbv = 1            # ENHANCEMENTS_FOR_SCHEDULED_TRAFFIC
+    Qbu = 2            # STRICT_PRIORITY
```

Comment:
As mentioned above, use the same names as in the ticket and rename TxCapability by TxSelection.

Qbu is part of the preemption support, unrelated to STRICT_PRIORITY. Please use the fields defined in the ticket.

Why do we need to define none?